aws-iot-mitigationaction/src/main/java/com/amazonaws/iot/mitigationaction/Translator.java

```diff
@@ -0,0 +1,202 @@
+package com.amazonaws.iot.mitigationaction;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import software.amazon.awssdk.services.iot.model.InternalFailureException;
+import software.amazon.awssdk.services.iot.model.InvalidRequestException;
+import software.amazon.awssdk.services.iot.model.IotException;
+import software.amazon.awssdk.services.iot.model.LimitExceededException;
+import software.amazon.awssdk.services.iot.model.MitigationActionParams;
+import software.amazon.awssdk.services.iot.model.ResourceAlreadyExistsException;
+import software.amazon.awssdk.services.iot.model.ResourceNotFoundException;
+import software.amazon.awssdk.services.iot.model.Tag;
+import software.amazon.awssdk.services.iot.model.ThrottlingException;
+import software.amazon.awssdk.services.iot.model.UnauthorizedException;
+import software.amazon.cloudformation.exceptions.BaseHandlerException;
+import software.amazon.cloudformation.exceptions.CfnAccessDeniedException;
+import software.amazon.cloudformation.exceptions.CfnAlreadyExistsException;
+import software.amazon.cloudformation.exceptions.CfnInternalFailureException;
+import software.amazon.cloudformation.exceptions.CfnInvalidRequestException;
+import software.amazon.cloudformation.exceptions.CfnNotFoundException;
+import software.amazon.cloudformation.exceptions.CfnServiceLimitExceededException;
+import software.amazon.cloudformation.exceptions.CfnThrottlingException;
+
+/**
+ * This class is a centralized placeholder for
+ *  - api request construction
+ *  - object translation to/from aws sdk
+ *  - resource model construction for read/list handlers
+ */
+
```

Comment:
Minor: There seem to be some extra new lines and inconsistent indentation depths. Let's reformat all the files that need it.

Reply:
Fixed